pkg/api/types.go

```diff
@@ -2428,6 +2428,13 @@ type PodSecurityContext struct {
 	// for that container.
 	// +optional
 	RunAsUser *int64
+	// The GID to run the entrypoint of the container process.
+	// Defaults to group specified in image metadata if unspecified.
+	// May also be set in SecurityContext.  If set in both SecurityContext and
+	// PodSecurityContext, the value specified in SecurityContext takes precedence
+	// for that container.
```

Comment:
@pweil- Do we really need to mention in each field that "If set in both SecurityContext and PodSecurityContext, the value specified in SecurityContext takes precedence for that container"? It makes description longer and also duplicates the same warning on the `PodSecurityContext` type.

Reply:
it's part of api documentation that is specific to the field.  I don't think it's wrong to have it.